spec/requests/api/v1/move_events_controller_create_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::MoveEventsController do
+  include ActiveJob::TestHelper
```

Comment:
Is this necessary?

Reply:
it isn't, now removed - too much copy pasta